docs/source/endpoints/relations.md

```diff
@@ -0,0 +1,246 @@
+---
+myst:
+  html_meta:
+    'description': 'Create, query, and delete relations between content items with the /@relations endpoint.'
+    'property=og:description': 'Create, query, and delete relations between content items with the /@relations endpoint.'
+    'property=og:title': 'Relations'
+    'keywords': 'Plone, plone.restapi, REST, API, relations, service, endpoint'
+---
```

Comment:
All the above should be double-quoted (`"` not `'`) for consistency with the Python script that generates them and the other files.